stream-core/src/main/java/org/apache/kylin/stream/core/util/NodeUtil.java

```diff
@@ -55,4 +71,29 @@ private static String getLocalhostName() {
         }
         return host;
     }
+
+    private static String getLocalHostIp() {
```

Comment:
Hi, from the issue https://issues.apache.org/jira/browse/KYLIN-3747,  the author MaGang recommend to use FQDN other than ip to register receiver. So, in my view, `getLocalHostIp` is not necessary, use `getLocalhostName` maybe better.

Reply:
Thank you for your reply. I'm in favor of using host name instead of IP.